### ISSUE TITLE
Task80:  Add new Subsite "Game Journals"

### ISSUE THREAD

**Issue by chris-kesler** [chris-kesler/KnowOnesWebsiteOfEverything#80]
Create a new subsite called game journals where I log my experiences as a DnD player, Dungeon Master, Minecraft player, Witcher 3 player, and poketwo player.

**Comment by chris-kesler** [comment]
Ruben Maribor - Witcher

Ruben was taken as a boy to become a Witcher.  Ruben passed the trials and became known as one of the best swordsman in the realm, both due to his skill and his unconventional left-handed reverse-grip.  

One fateful encounter left Ruben with Wyvern poison coursing through his left forearm.  He managed to survive,

**Comment by chris-kesler** [comment]
Ruben Jarricor - Post Session One

I awoke tied down upon a stone altar.  My mouth was gagged and my head covered with a sack so I could not see beyond my immediate surroundings.  My captor seemed to be male, and took a great amount of pleasure in wounding me.  I was left alone after some time with lacerations and bruises, but I was still in capable condition.  Yelling for a time seemed to do little to improve my situation until a group of five strangers entered.  A drow woman quickly cut my bindings, and I thanked her.  She seemed to be light on her feet and quick with a blade. 
The rest of the group comprised of a male Orcish Blood Hunter wielding a Warhammer, a shy female Tiefling Bard, a male Elven Ranger who scours his surroundings like a hound, and a female Elven Cleric who seems perpetually unimpressed with her surroundings.  Most ignored my presence and looted the room shamelessly.  It's clear that they are mercenaries, bandits, or plunderers.  

I offered to help them explore the remainder of this crypt as a thanks for freeing me.  I was able to convince them of my capabilities, and they accepted me without much consideration.  I'm glad to have allies, but in truth, I'm simply grateful they didn't try to kill me and loot as they seem so inclined on doing.  

We descended from the elevated room where I was being kept using some of my rope, and continued to the left into an open room.  Several Kobolds were attempting to open a cell door, but we quickly slew them when they became hostile.  They were hell bemt on looting the cell, so I helped the elven woman who freed me pick the lock.  Inside were several coins and a valuable ring.  

We continued to the right through a hallway.  I was able to spot and disable a rather nasty trap, further cementing my value as a member with these plunderers.  The following room contained a statue with a riddle regarding the authenticity of gold coins.  The left hand held 5, the right 4, yet they were balanced as though there were four in each on a merchant scale.  I suspect that one coin is either heavier or lighter due to a core of differing value and that the balance is symbolic of currency value.  I suggested we wait and address this riddle after clearing the rest of the crypt.  

We continued forward, finding a locked door to the right.  We were unable to pick the lock, so I peered through the keyhole using a lit ball bearing.  Inside was a chest, which I intend to come back for.  I may call these people plunderers, but I am not one to ignore resources.  

We continued to the next room, which was styled after the inside of a cathedral, and found more Kobolds.  They were dispatched quickly, though their leader did put up quite a fight.  I found several healing potions in a strongbox at the back of the cathedral room where the leader was lurking.  There appears to be a flight of stairs beyond as I write this.  We will see where they lead after we are all rested and recovered.
